server/middleware/auth.js

```diff
@@ -0,0 +1,13 @@
+const jwt = require('jsonwebtoken')
+
+module.exports = function(req, res, next) {
+    const authToken = req.header("user-auth-token");
+
+    if (!authToken) {
+        return res.status(401).send('User not authorized!')
+    }
+
+    const userDetails = jwt.verify(authToken, 'awognawole3280843halegn');
```

Comment:
`jwt.verify` could throw an error and crash the server. E.g: a user sending a payload with invalid content *and* format.

Let's make sure we handle that case on the middleware.